src/main/java/eu/ai4work/sws/model/generic/SlidingDecisionStatus.java

```diff
@@ -0,0 +1,16 @@
+package eu.ai4work.sws.model.generic;
+
+public enum SlidingDecisionStatus {
+    REQUEST("Sliding Decision Request"),
+    RESPONSE("Sliding Decision Response");
+
+    private final String displayName;
+
+    SlidingDecisionStatus(String displayName) {
+        this.displayName = displayName;
+    }
+
+    public String getDisplayName() {
```

Comment:
Related to the other suggestions for "enum instead of String", I would suggest to use the `JsonValue` annotation here so that the `displayName` is used in the resulting JSON:
```suggestion
    @JsonValue
    public String getDisplayName() {
```